src/runtime_src/core/include/xgq.h

```diff
@@ -124,7 +134,35 @@ struct xrt_com_queue_entry {
 };
 
 #define XGQ_SUB_HEADER_SIZE	(sizeof(struct xrt_sub_queue_entry) - 4)
-#define	XRT_COM_Q1_SLOT_SIZE	(sizeof(struct xrt_com_queue_entry))
+#define XRT_COM_Q1_SLOT_SIZE	(sizeof(struct xrt_com_queue_entry))
```

Comment:
warning: macro `XRT_COM_Q1_SLOT_SIZE` used to declare a constant; consider using a `constexpr` constant [cppcoreguidelines-macro-usage]

```cpp
#define XRT_COM_Q1_SLOT_SIZE    (sizeof(struct xrt_com_queue_entry))
        ^
```

Reply:
Use `// NOLINT `if this is not a C++ header and the #defines are needed.